docs/docs/developers/quickstart.md

```diff
@@ -49,3 +52,67 @@ To copy a scene in Unity:
 
 With just these steps, you can immediately build and release your own customized Digital Twin; next up you can read up 
 on what and how you can configure the platform without making any code changes.
+
+
+
+
+## Development on macOS
+
+> ⚠️ **Note:** Microsoft ended support for Visual Studio for Mac on **August 31, 2024**. The final version (17.6) is now considered a legacy installation.
+
+### Recommended Development Options on macOS
+
+If you're working on macOS with Unity and C#, we recommend one of the following setups:
+
+- **Visual Studio Code** with the [C# Dev Kit](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csdevkit) and Unity extensions  
+  → Lightweight, free, and actively maintained.
+
+- **JetBrains Rider**  
+  → A commercial IDE with excellent Unity and C# support. Subscription required.
+
+- **Visual Studio on Windows**  
+  → Can be run via a virtual machine or through [Microsoft Dev Box](https://azure.microsoft.com/en-us/products/dev-box) (cloud-based).
+
+Currently, **Visual Studio Code with extensions** is the most accessible and supported solution for Unity development on macOS.
```

Comment:
Is dit een mening of echt waar? Want als het meer een mening is, dan kunnen we het beter weglaten.